devel/spicy/getting-started.rst

```diff
@@ -91,6 +91,21 @@ having the expected signature of four parameters matching the types of
 the parameter expressions that the ``*.evt`` file specifies. The
 handler's body then just prints out what it gets.
 
+Further, you see a ``zeek_init()`` event handler that calls the Analyzer
+framework's ``register_for_port()`` function. This call tells Zeek to
+enable the ``spicy::MyHTTP`` analyzer for connections on TCP port 12345.
+
+.. note::
+
+  .. versionchanged:: 7.1
```

Comment:
I'm torn on this: we don't use `versionchanged` anywhere else but usually just update the docs and otherwise rely on NEWS. But I see how this can be helpful of course, and maybe we should start doing this. So ok to keep I guess. :-)

Reply:
I've used `versionadded` a bit in the Zeek docs as it's nice for people that look at docs.zeek.org "master" version online to go "hmm, which version are we running, actually?" and "hmm, we should upgrade" nudge :-)

![image](https://github.com/user-attachments/assets/5207358a-841c-4d60-b4eb-3a61e0fbd296)